sdk/include/psdk/versionhelpers.h

```diff
@@ -145,12 +145,32 @@ IsActiveSessionCountLimited()
 }
 
 #ifdef __REACTOS__
-VERSIONHELPERAPI
-IsReactOS()
-{
-    // FIXME: Find a better method!
-    WCHAR szWinDir[MAX_PATH];
-    GetWindowsDirectoryW(szWinDir, _countof(szWinDir));
-    return (wcsstr(szWinDir, L"ReactOS") != NULL);
+#include <winreg.h>
+typedef DWORD (WINAPI FN_SHGetValueW(HKEY hKey, LPCWSTR lpszSubKey, LPCWSTR lpszValue,
```

Comment:
This "leaks" the function type to global scope. Declare it inside the function...